docs_src/microservices/support/cronScheduler/Configuration.md

```diff
@@ -0,0 +1,39 @@
+--- 
+title: Support Cron Scheduler - Configuration
+---
+
+# Support Cron Scheduler - Configuration
+
+Please refer to the general [Common Configuration documentation](../../configuration/CommonConfiguration.md) for configuration settings common to all services.
+Below are only the additional settings and sections that are specific to Support Cron Scheduler.
+
+!!! edgey "Edgex 3.0"
+    For EdgeX 3.0 the `MessageQueue` configuration has been moved to `MessageBus` in [Common Configuration](../../../configuration/CommonConfiguration/#configuration-properties)
+
+=== "Writable"
+    |Property|Default Value|Description|
+    |---|---|---|
+    ||Writable properties can be set and will dynamically take effect without service restart|
+    |LogLevel|INFO|log entry [severity level](https://en.wikipedia.org/wiki/Syslog#Severity_level).  Log entries not of the default level or higher are ignored. |
+=== "Writable.Telemetry"
+    |Property|Default Value|Description|
+    |---|---|---|
+    |||See `Writable.Telemetry` at [Common Configuration](../../../configuration/CommonConfiguration/#configuration-properties) for the Telemetry configuration common to all services |
+    |Metrics| `TBD` |Service metrics that Support Cron Scheduler collects. Boolean value indicates if reporting of the metric is enabled.|
+    |Tags|`<empty>`|List of arbitrary service level tags to included with every metric that is reported. i.e. `Gateway="my-iot-gateway"` |
+=== "Service"
+    |Property|Default Value|Description|
+    |---|---|---|
+    ||| Unique settings for Support Cron Scheduler. The common settings can be found at [Common Configuration](../../../configuration/CommonConfiguration/#configuration-properties)
+    |Port|59863|Micro service port number|
+    |StartupMsg |This is the Support Cron Scheduler Microservice|Message logged when service completes bootstrap start-up|
+=== "MessageBus.Optional"
+    |Property|Default Value|Description|
+    |---|---|---|
+    ||| Unique settings for Support Cron Scheduler. The common settings can be found at [Common Configuration](../../../configuration/CommonConfiguration/#configuration-properties)
+    |ClientId|"support-cron-scheduler| Id used when connecting to MQTT or NATS base MessageBus |
+
```

Comment:
all the private config should be included in this table, including Database
https://github.com/edgexfoundry/edgex-go/blob/872e8ffdf3f0ef6c4a8f4e5a20969148dd3ced41/cmd/support-cron-scheduler/res/configuration.yaml#L28

Add a note to state this service only supports postgres as the persistence layer for this release